apps/vaporgui/FlowSubtabs.cpp

```diff
@@ -119,9 +129,61 @@ void FlowVariablesSubtab::_dimensionalityChanged( int nDims ) const {
 FlowAppearanceSubtab::FlowAppearanceSubtab(QWidget* parent) : QVaporSubtab(parent)
 {
     _TFEditor = new TFEditor(true);
+    _pw = new PGroup;
 
     _layout->addWidget( _TFEditor, 0, 0 );
-
+    _layout->addWidget(_pw);
+    
+    PSection *ps;
+    
+    _pw->Add(ps = new PSection("Appearance"));
+    ps->Add(new PEnumDropdown(FlowParams::RenderTypeTag, {"Tubes", "Samples"}, {FlowParams::RenderTypeStream, FlowParams::RenderTypeSamples}, "Render Type"));
```

Comment:
It looks like this will bring more low-level Params calls into the code base.  Do we want to fix this now or later?  It's hard to say now because the dust hasn't quite settled regarding PWidget usage.  I'd be ok with fixing this later, but I'm also inclined to steer away from this if we truly want the low-level interface to be inaccessible from the GUI.

Reply:
Let's talk about this during the meeting